Doc/library/sys.rst

```diff
@@ -303,6 +303,10 @@ always available.
    the real path to its executable, :data:`sys.executable` will be an empty string
    or ``None``.
 
+   .. versionchanged:: 3.7.1
+      Included executable suffix to path, if one is defined for the
+      platform, unless the executable path is a symbolic link.
```

Comment:
EXE_SUFFIX is specified by configure option.  So "defined for the platform" is bit unclear.
Especially, suffix is added for Windows build already before this change.